genomic-medicine-sweden/JASEN
# Self-test preliminaries

# Description
This PR was prompted as users were unable to install and run the pipeline. 

_Summary of the changes made:_

- Rewrote the usage instructions under README
- Added a base config for self tests
- Added dedicated refseq downloader
- Added reference downloader for various resources
- Linked in various databases and other resources
- Added a logotype
- Created an example command to check for installation stability
- Added example sample metadata file
- Added index file to know what to put into a conda env
- Added requirements file to quickly build conda env
- Added setup script to easily create conda environment
- Moved folders around

## Primary function of PR
- [ ] Major functionality improvement / New type of analysis
- [ ] Backward-breaking functionality

# Testing
- Review the README instructions
- Create an installation
- Run the test command listed in the README
- Confirm the pipeline finishes with no unexpected errors

# Sign-offs
- [ ] Code reviewed by @octocat
